packages/valory/skills/safe_deployment_abci/behaviours.py

```diff
@@ -59,23 +59,20 @@ def synchronized_data(self) -> SynchronizedData:
 class RandomnessSafeBehaviour(RandomnessBehaviour):
     """Retrieve randomness for oracle deployment."""
 
-    behaviour_id = "randomness_safe"
```

Comment:
This shows the benefit. We can drop these - optionally.

Reply:
That would mean that we will need to update the tests accordingly.